### PR TITLE
[17.0][MIG] stock_packaging_calculator: Migration to 17.0

### PR DESCRIPTION
**Pull request by nguyenminhchien**

### CODE REVIEW

File: stock_packaging_calculator/models/product_qty_by_packaging_mixin.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+# @author: Simone Orsi <simone.orsi@camptocamp.com>
+# @author: Sébastien Alix <sebastien.alix@camptocamp.com>
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl)
+from odoo import api, fields, models
+
+
+class ProductQtyByPackagingMixin(models.AbstractModel):
+    """Allow displaying product qty by packaging."""
+
+    _name = "product.qty_by_packaging.mixin"
```
Comment (rousseldenis) [inline stock_packaging_calculator/models/product_qty_by_packaging_mixin.py:11]: Adding tests for the mixin should be great as it is a transversal feature.

File: stock_packaging_calculator/readme/ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+TODO
```
Comment (rousseldenis) [inline stock_packaging_calculator/readme/ROADMAP.md]: What about these?

File: stock_packaging_calculator/tests/common.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+class TestCommon(TransactionCase):
+    at_install = False
+    post_install = True
+    maxDiff = None
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```
Comment (rousseldenis) [inline stock_packaging_calculator/tests/common.py]: FYI, inheriting from BaseCommon avoid adding this line : https://github.com/odoo/odoo/blob/17.0/odoo/addons/base/tests/common.py#L21

File: stock_packaging_calculator/models/product_qty_by_packaging_mixin.py (hunk 1 above)
Comment (nguyenminhchien) [reply stock_packaging_calculator/models/product_qty_by_packaging_mixin.py:11]: The tests just added

File: stock_packaging_calculator/readme/ROADMAP.md (hunk 2 above)
Comment (nguyenminhchien) [reply stock_packaging_calculator/readme/ROADMAP.md]: i just fixed this issue. Please have a look.

**Review by rousseldenis — APPROVED** [review]
Code review. Seems great

**Review by leemannd — APPROVED** [review]

**Pull request merged**